Phonebook.Frontend/src/index.html

```diff
@@ -47,5 +46,39 @@
     } else {
       top.location = self.location;
     }
+    msieversion();
+
+    function openURL(e) {
+      browser = e.getAttribute('value');
+      if (browser == 'microsoft-edge') {
+        //edge can not be opened via shell
+        window.open((browser = e.getAttribute('value') + ':' + window.location.href));
+      } else {
+        new ActiveXObject('WScript.Shell').run(browser + ' ' + window.location.href);
+      }
+    }
+
+    function msieversion() {
+      var browserList = {
+        //list of browser-links where app can be openend in: key=browser name, value=shell-command to open browser
+        'Microsoft Edge': 'microsoft-edge',
+        'Google Chrome': 'chrome',
+        'Mozilla Firefox': 'firefox',
+        Opera: 'opera'
+      };
+      var ua = window.navigator.userAgent;
+      var msie = ua.indexOf('MSIE ');
+
+      if (msie > 0 || !!navigator.userAgent.match(/Trident.*rv\:11\./)) {
+        // stone age browser IE
+        var html =
+          '<div class="pb-no-ie"><p><strong>This application does not support Internet Explorer!</strong></p><p>To run the Phonebook, please use one of the following modern browsers:</p><ul>';
+        for (var key in browserList) {
+          html += '<li onclick="openURL(this)" value="' + browserList[key] + '"><strong>' + key + '</strong></li>';
+        }
+        html += '</ul></div>';

```

Comment:
```suggestion
        html += '</ul><p>If this was detected by mistake consider <a href="https://github.com/T-Systems-MMS/phonebook/issues/new/choose">opening an issue</a>.</p> </div>';

```

Reply:
Github also doesnt support ie, so if the user clicks the link, he will get a similar message :-D
![image](https://user-images.githubusercontent.com/55480780/76406560-37bd2200-638a-11ea-82eb-35cbd486704e.png)